README.md

```diff
@@ -1,19 +1,89 @@
+Remember not to do this manually - **whenever the user refreshes the page, we should query the API for a new deck and save our new deck ID.**
+
+### Drawing Cards
+
+The next thing we'd like to do is to draw cards from our deck.
+Using our deck ID, the Deck of Cards API lets us draw one or more cards from the deck that we just created.
+The route for drawing five cards should look something like this: `https://deckofcardsapi.com/api/deck/~deckId~/draw/?count=5`, where `~deckId~` is the deck ID item we saved.
+Use Axios to query this route.
+
+What is available to us here in the response? Well, again, we have only one key in this response that we really care about - `cards`, which contains an array of `Card` objects.
+Each of these `Card` objects have a set of parameters, all of which could prove useful to us, depending on what we want to do with them: There's an `image` URL, which links to a picture of the card.
+There's a `value` and `suit` parameter, and then there's a `code` parameter with shorthand should we need it.
+
+Create a `button` tag in your HTML.
+When you click on this `button`, you should fire your Axios request to draw five cards.
+Save them to a variable in your project.
+When you click the button again, you should draw five different cards.
+
+### Rendering Cards to the DOM
+
+When we click, we should see the images of five cards, in a row, underneath our button.
+This should be familiar to you from our Dog API image rendering.
+In your button event listener, after your Axios request, loop through your five cards.
+Create `img` tags for each of them, give each a class of `.image`, set each `src` to a the card's image URL, and append them to the DOM underneath your `button` tag.
+
+Just like our Dog API project, when you click the button again, it should remove/replace each card with a new card.
+
+### Drawing More or Fewer Cards
+
+Create a `select` tag underneath your `button`.
+Your `select` tag should be filled with numbers from 1 through 10.
+Give it a default value of 5 to start.
+
+### Styling
+
+It's time to make your browser look like a genuine card table.
+On your `body`, change the background color to be `green`.
+Create a `div` tag to surround your `button`, `select` box, and cards, and make the background color `darkgreen`.
+Center your cards and inputs on the page, and give them some `margin` of `10px` so that they aren't touching the top of the page directly.
+
+Finally, add a `hover` effect to each card image to `transform` the card to a `1.1` scale when the user hovers over it - as if they're selecting that card.
+Isn't it satisfying to mouse over all your cards now?
 
 ## Submission Guidelines
-  * When finished, commit and push your work.
-  * Make a pull request on github.
-  * Submit the link to your pull request on Canvas.
 
-[Additional instructions]
\ No newline at end of file
+- When finished, commit and push your work.
+- Make a pull request on github.
+- Submit the link to your pull request on Canvas.
```

Comment:
Do you think you can add a note or small section explaining that these tests use fixtures? 
- what are fixtures
- where to find the fixtures
- tips for working with test output when there are fixtures

Reply:
I'm not 100% that what I wrote will completely satisfy what you're looking for... but it's a start I suppose 😄